lib/utils/cloud.ts

```diff
@@ -106,3 +106,105 @@ export const getDeviceAndMaybeAppFromUUID = _.memoize(
 	// Memoize the call based on UUID
 	(_sdk, deviceUUID) => deviceUUID,
 );
+
+/**
+ * Download balenaOS image for the specified `deviceType`.
+ * `OSVersion` may be one of:
+ *  - exact version number,
+ *  - valid semver range,
+ *  - `latest` (includes pre-releases),
+ *  - `default` (excludes pre-releases if at  least one stable version is available),
+ *  - `recommended` (excludes pre-releases, will fail if only pre-release versions are available),
+ *  - `menu` (will show the interactive menu )
+ * If not provided, OSVersion will be set to `default`
+ *
+ * @param deviceType
+ * @param outputPath
+ * @param OSVersion
+ */
+export async function downloadOSImage(
+	deviceType: string,
+	outputPath: string,
+	OSVersion?: string,
```

Comment:
How about declaring this parameter with a default value?
```
	OSVersion = 'default',
```

The reason for this suggestion is that I saw the constant `'default'` being defined in `actions-oclif/device/init.ts` and also below:
```
	OSVersion = OSVersion
		? await resolveOSVersion(deviceType, OSVersion)
		: 'default';
```
It's nicer to have the constant in a single place. But we'd need to check that a blank string is not provided as argument, as a blank string currently gets detected by the `if (!OSVersion)` test but would not be replaced by a default value here.

Reply:
Pro's and cons I think.  With the current setup the caller (either a user of the os download command, or some part of the CLI), can explicitly specify `default` as the version supress the warning (which seems reasonable).

Reply:
Though considering the message is nice and short now, there's no harm in always printing it I suppose.

Reply:
> can explicitly specify default as the version suppress the warning

True. I think `OSVersion?: string` was better after all, considering that a user might type `'default'` on the command line.